app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -26,6 +26,8 @@ def button
     # Handle Toolbar Policy Tag Button
     @refresh_div = "main_div" # Default div for button.rjs to refresh
     tag(self.class.model) if params[:pressed] == "#{params[:controller]}_tag"
+    assign_policies(ContainerImage) if params[:pressed] == "container_image_protect"
+    check_compliance_images if params[:pressed] == "container_image_check_compliance"
```

Comment:
@jrafanie I can submit a patch (at least partial since this is all over the place) but I need to understand better what you want. Isn't this(params[:pressed]) a url param passed from the client[1]? So this class should maintain a static list of some object that contains methods for operations?

[1] I was never able to find what server component populates params and edit parameters